assets/style.scss

```diff
@@ -110,23 +110,20 @@ p {
 
 ul {
   list-style: none;
-}
-li {
-  font-size: 17px;
-  font-weight: normal;
-  color: var(--text-color);
-}
-.list::before {
-  content: "";
-  width: 7px;
-  height: 7px;
-  margin-right: calc(var(--margin) - 5px);
-  background-color: var(--deep-purple);
-  border-radius: 100%;
-  display: inline-block;
-}
-.list p {
-  margin-left:22px;
+  li {
+    font-size: 17px;
+    font-weight: normal;
+    color: var(--text-color);
+    margin-left: 1em;
+    &:before {
```

Comment:
why keep using the `:before` and not the `list-style` like we discussed?

Reply:
Changed my mind and switched for list-style